🤖/bitcoinbot.yml

```diff
@@ -0,0 +1,26 @@
+
+added	/trunk
+added	/trunk/base58.h
+added	/trunk/bignum.h
+added	/trunk/db.cpp
+added	/trunk/db.h
+added	/trunk/headers.h
+added	/trunk/irc.cpp
+added	/trunk/irc.h
+added	/trunk/key.h
+added	/trunk/libeay32.dll
+added	/trunk/license.txt
+added	/trunk/main.cpp
+added	/trunk/main.h
+added	/trunk/makefile
+added	/trunk/makefile.vc
```

Comment:
{
  "language": "shell",
  "os": [
    "linux"
  ],
  "dist": "bionic",
  "version": "~> 1.0",
  "arch": [
    "amd64"
  ],
  "cache": {
    "directories": [
      "$TRAVIS_BUILD_DIR/depends/built",
      "$TRAVIS_BUILD_DIR/depends/sdk-sources",
      "$TRAVIS_BUILD_DIR/ci/scratch/.ccache",
      "$TRAVIS_BUILD_DIR/releases/$HOST"
    ]
  },
  "stages": [
    {
      "name": "lint"
    },
    {
      "name": "test"
    }
  ],
  "env": {
    "global": [
      {
        "CI_RETRY_EXE": "\"travis_retry\""
      },
      {
        "CACHE_ERR_MSG": "\"Error! Initial build successful, but not enough time remains to run later build stages and tests. See https://docs.travis-ci.com/user/customizing-the-build#build-timeouts . Please manually re-run this job by using the travis restart button. The next run should not time out because the build cache has been saved.\""
      }
    ]
  },
  "before_install": [
    "set -o errexit; source ./ci/test/00_setup_env.sh",
    "set -o errexit; source ./ci/test/03_before_install.sh"
  ],
  "install": [
    "set -o errexit; source ./ci/test/04_install.sh"
  ],
  "before_script": [
    "for i in {1..4}; do echo \"$(sleep 500)\" ; done &",
    "set -o errexit; source ./ci/test/05_before_script.sh &> \"/dev/null\""
  ],
  "script": [
    "export CONTINUE=1",
    "if [ $SECONDS -gt 1200 ]; then export CONTINUE=0; fi",
    "if [ $TRAVIS_REPO_SLUG = \"bitcoin/bitcoin\" ]; then export CONTINUE=1; fi",
    "if [ $CONTINUE = \"1\" ]; then set -o errexit; source ./ci/test/06_script_a.sh; else set +o errexit; echo \"$CACHE_ERR_MSG\"; false; fi",
    "if [[ $SECONDS -gt 50*60-$EXPECTED_TESTS_DURATION_IN_SECONDS ]]; then export CONTINUE=0; fi",
    "if [ $TRAVIS_REPO_SLUG = \"bitcoin/bitcoin\" ]; then export CONTINUE=1; fi",
    "if [ $CONTINUE = \"1\" ]; then set -o errexit; source ./ci/test/06_script_b.sh; else set +o errexit; echo \"$CACHE_ERR_MSG\"; false; fi"
  ],
  "after_script": [
    "echo $TRAVIS_COMMIT_RANGE"
  ],
  "jobs": {
    "include": [
      {
        "stage": "lint",
        "name": "lint",
        "env": [
          {}
        ],
        "cache": {
          "pip": true
        },
        "language": "python",
        "python": "3.6",
        "install": [
          "set -o errexit; source ./ci/lint/04_install.sh"
        ],
        "before_script": [
          "set -o errexit; source ./ci/lint/05_before_script.sh"
        ],
        "script": [
          "set -o errexit; source ./ci/lint/06_script.sh"
        ]
      }
    ]
  }
}